frontend/src/poc-code/testK8s/WSTest.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+import * as React from 'react';
+import { WSFactory } from './ws-factory';
+import PrintObject from './PrintObject';
+import { Alert, Button, Spinner, Title } from '@patternfly/react-core';
+
+const getToken = (): string | null => {
+  const cookieToken = document.cookie.split('; ').find((val) => val.startsWith('cs_jwt='));
+  if (!cookieToken) {
+    // Promise.reject('Could not make k8s call. Unable to find token.');
+    console.error('no token??', typeof cookieToken, cookieToken);
+    return null;
+  }
+  const [, token] = cookieToken.split('=');
+
+  return token;
+};
+const base64 = (data: string): string => {
+  // console.debug('equals??', btoa(data).replace(/\+/g, '-').replace(/\//g, '_').split('=', 1)[0]);
+  return btoa(data).replace(/\+/g, '-').replace(/\//g, '_').split('=', 1)[0];
+};
+
+const WSTest = () => {
+  const [r, setR] = React.useState(null);
+  const [createSocket, setCreateSocket] = React.useState(false);
+  const [error, setError] = React.useState(null);
+
+  // To get WS to work even a bit, we need to modify @redhat-cloud-services/frontend-components-config-utilities
+  // in 'frontend/node_modules/@redhat-cloud-services/frontend-components-config-utilities/repos/insights-chrome-build/snippets/head.html'
+  // 'wss:' needs to exist next to 'https:'
```

Comment:
@karelhala I'll need your help to where to put up a PR for this and getting approvals for it.

Reply:
Woot! Thanks. I see it live, I'll remove this comment when I update my PR on top of what you just merged.